src/Main.cpp

```diff
@@ -48,6 +52,56 @@ struct MovementSystem
 			F32 fx  = fnx+movement.force.x;             /* Total force. */
 			F32 fy  = fny+movement.force.y;
 
+			for (auto&& [entity2, movement2, pos2] : access_storage().each())
+			{
+				F32 dx, dy;
+
+				if (entity <= entity2)
+				{
+					continue;
+				}
+
+				dx = pos2.xy.x - pos.xy.x;
+				dy = pos2.xy.y - pos.xy.y;
+				if (fabsf(dx) < (movement.collision_box_width + movement2.collision_box_width) / 2.0f &&
```

Comment:
Use a variable for `movement.collision_box_height + movement2.collision_box_height` and similarly for `width` to make this code more readable.

Reply:
Here and everywhere below